README.md

```diff
@@ -27,30 +21,37 @@
 
 </div>
 
-## Documentation
+[Material UI](https://mui.com/material-ui/) is a comprehensive library of React components that features our independent implementation of Google's [Material Design](https://m2.material.io/design/introduction/) system.
+It's trusted by some of the world's greatest product teams because it's been rigorously battle-tested through more than a decade of development by thousands of open-source contributors.
 
-### Material UI
+Material UI's core functionality is extended by [MUI X](https://github.com/mui/mui-x), a suite of complex components for advanced use cases.
```

Comment:
I wonder if it wouldn't make sense to extend this a bit more. I could see use explaining Material UI as a library but also a stack, like shadcn.

In v6, this stack is only MUI System + the components. 

In v7, this stack could look like this:

- Base UI (@base-ui/react)
- Material UI System (@material-ui/system), which is a slim wrapper on Pigment CSS, where Pigment CSS supports a runtime with emotion too.
- Pre-designed components following Material Design built on top of those (@material-ui/md3), this package can be in anyone's codebase.